edpcmentoring/edpcmentoring/settings.py

```diff
@@ -6,6 +6,11 @@
 
 """
 
+try:
```

Comment:
I think this is doing it slightly backwards. I think it's more obvious to have a ``settings_local`` (or ``settings_mws``) which imports ``settings`` and the overrides rather than having ``settings`` try to determine if some settings are already present.